docs/attributes-registry/user.md

```diff
@@ -10,13 +10,16 @@
+| ------------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- | ---------------------------------------------------------------- |
+| `user.authentication.id` | string   | Unique identifier of an authenticated user in the system.                                                                                                                           | `S-1-5-21-202424912787-2692429404-2351956786-1000` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.email`             | string   | User email address.                                                                                                                                                                 | `a.einstein@example.com`                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.full_name`         | string   | User's full name                                                                                                                                                                    | `Albert Einstein`                                  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.hash`              | string   | Unique user hash to correlate information for a user in anonymized form. [1]                                                                                                        | `364fc68eaf4c8acec74a4e52d7d1feaa`                 | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.id`                | string   | Identifier of a user who interacts with a system. This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system. [2] | `QdH5CAWJgqVT4rOr0qtumf`                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.name`              | string   | Short name or login/username of the user.                                                                                                                                           | `a.einstein`                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `user.roles`             | string[] | Array of user roles at the time of the event.                                                                                                                                       | `["admin", "reporting_user"]`                      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
 **[1]:** Useful if `user.id` or `user.name` contain confidential information and cannot be used.
```

Comment:
we probably want to update this note to include `user.authenticated.id`.

But also given  that `user.id` is now random and not-authenticated one, we probably can remove the assumption that it may contain sensitive information.

Reply:
i've updated it to 'user.authentication.id'